### PR TITLE
Clean up attributes for logs/traces and user id correlation

### PR DESCRIPTION
**Pull request by adamhathcock**

### CODE REVIEW

File: Sdk/Speckle.Connectors.Common/Connector.cs [hunk 1]
```diff
@@ -24,8 +24,8 @@ HostAppVersion version
     TypeLoader.Initialize(typeof(Base).Assembly, typeof(Point).Assembly);
 
     var (logging, tracing) = Observability.Initialize(
+      application.Name + " " + HostApplications.GetVersion(version),
```
Comment (oguzhankoral) [inline Sdk/Speckle.Connectors.Common/Connector.cs:27]: changing the order of properties was intentional or not?

Comment (adamhathcock) [reply Sdk/Speckle.Connectors.Common/Connector.cs:27]: Intentional in that I just put would I thought was most significant to least.  But I did go through some refactorings

File: Sdk/Speckle.Connectors.Logging/Internal/LogBuilder.cs [hunk 2]
```diff
@@ -18,16 +15,9 @@ public static LoggerProvider Initialize(
     ResourceBuilder resourceBuilder
   )
   {
-    var fileVersionInfo = GetFileVersionInfo();
     var serilogLogConfiguration = new LoggerConfiguration()
       .MinimumLevel.Is(SpeckleLogLevelUtility.GetLevel(speckleLogging?.MinimumLevel ?? SpeckleLogLevel.Warning))
       .Enrich.FromLogContext()
-      .Enrich.WithProperty("version", fileVersionInfo.FileVersion)
-      .Enrich.WithProperty("productVersion", connectorVersion)
-      .Enrich.WithProperty("hostOs", DetermineHostOsSlug())
```
Comment (oguzhankoral) [inline Sdk/Speckle.Connectors.Logging/Internal/LogBuilder.cs:27]: hmm, why don't we?

Comment (adamhathcock) [reply Sdk/Speckle.Connectors.Logging/Internal/LogBuilder.cs:27]: this is now part of tags and will attach to all traces and logs as part of Otel

**Review by AlanRynne — APPROVED** [review]

**Pull request merged**